libraries/ESP32/examples/Serial/Interrupt/Interrupt.ino

```diff
@@ -0,0 +1,21 @@
+HardwareSerial hwSerial(0);
+HardwareSerial hwSerial2(2);
```

Comment:
This also exists as `Serial2`, though I will recommend you to use `Serial1` instead, because some newer chips like S2 and C3 do not have third UART port.

Reply:
But on those chips, will compiling Serial2 fail? Or will it just use Serial?

Reply:
on those chips `Serial` and `Serial1` exists. `Serial2` or `HardwareSerial hwSerial2(2)` do not and will cause error.

Reply:
I'm changing the example to something else!